i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/operatingsystems-linux-ssh.md

```diff
@@ -218,166 +333,1027 @@ Ce connecteur de supervision est compatible avec n'importe quelle distribution L
+| CRITICALUSAGEPRCT | Critical threshold                                                                                  |                   |             |
+| EXTRAOPTIONS      | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) | --verbose         |             |
 
-> Avec ce backend, il est nécessaire d'effectuer une connexion manuelle entre l'utilisateur centreon-engine du Collecteur
-et l'utilisateur applicatif créé sur le serveur Linux. (Macro SSHUSERNAME).
+</TabItem>
+<TabItem value="Swap" label="Swap">
+
+| Macro             | Description                                                                                         | Valeur par défaut            | Obligatoire |
+|:------------------|:----------------------------------------------------------------------------------------------------|:-----------------------------|:-----------:|
+| WARNINGUSAGE      | Threshold, can be 'usage' (in Bytes), 'usage-free' (in Bytes), 'usage-prct' (%)                     |                              |             |
+| CRITICALUSAGE     | Threshold, can be 'usage' (in Bytes), 'usage-free' (in Bytes), 'usage-prct' (%)                     |                              |             |
+| WARNINGUSAGEFREE  | Threshold, can be 'usage' (in Bytes), 'usage-free' (in Bytes), 'usage-prct' (%)                     |                              |             |
+| CRITICALUSAGEFREE | Threshold, can be 'usage' (in Bytes), 'usage-free' (in Bytes), 'usage-prct' (%)                     |                              |             |
+| WARNINGUSAGEPRCT  | Threshold, can be 'usage' (in Bytes), 'usage-free' (in Bytes), 'usage-prct' (%)                     |                              |             |
+| CRITICALUSAGEPRCT | Threshold, can be 'usage' (in Bytes), 'usage-free' (in Bytes), 'usage-prct' (%)                     |                              |             |
+| EXTRAOPTIONS      | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) | --verbose --use-new-perfdata |             |
 
 </TabItem>
-<TabItem value="libssh backend (par défaut)" label="libssh backend (par défaut)">
+<TabItem value="Systemd-Journal" label="Systemd-Journal">
+
+| Macro           | Description                                                                                                                                                     | Valeur par défaut | Obligatoire |
+|:----------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| SINCE           | Defines the amount of time to look back at messages. Can beminutes (ie 5 "minutes ago") or 'cache' to use the timestamp from last execution. (Default: 'cache') | cache             |             |
```

Comment:
```suggestion
| SINCE           | Defines the amount of time to look back at messages. Can be minutes (i.e. 5 "minutes ago") or 'cache' to use the timestamp from last execution. (Default: 'cache') | cache             |             |
```